cmd/preflight/cmd/root.go

```diff
@@ -48,7 +51,13 @@ func rootCmd() *cobra.Command {
 }
 
 func Execute() error {
-	return rootCmd().ExecuteContext(context.Background())
+	zapLog, err := zap.NewDevelopment()
```

Comment:
Is this supposed to be `zap.NewProduction()`? I'm not super familiar with Zap.

Reply:
It's just PoC. This will absolutely change to set up logging how preflight cli wants it.